projects/iio_demo/Makefile

```diff
@@ -1,7 +1,10 @@
-TARGET := iio_demo
-TINYIIOD ?= y
```

Comment:
Would you not still need this line to build the tinyiiod library since there is still [this line](https://github.com/analogdevicesinc/no-OS/blob/dc8b51f02864eaa74c1aa171effe015b275a3ab3/tools/scripts/linux.mk#L398) in the code?

Reply:
no, iio_demo uses LIBRARIES += iio

Reply:
and in the previous commit is the implementation for the LIBRARIES variable